### PR TITLE
add typing-extensions to list of dependencies

### PR DESCRIPTION
**Pull request by biamalveiro**
Hey! 😄 

I installed the theme with`pip install feedzai-altair-theme` and when I tried to enable it with `alt.themes.enable("feedzai")` I got an error`ModuleNotFoundError: No module named 'typing-extensions`. Seems that it is need in `types.py` and we are missing it in the dependencies in `setup.py`.

https://github.com/feedzai/feedzai-altair-theme/blob/529618a250ae57269df69e1226639cbc26376002/feedzai_altair_theme/types.py#L20

This pull request would just add it to the list of dependencies 📦

### CODE REVIEW

**Review by joaopalmeiro — CHANGES_REQUESTED** [review]
Thank you for this, @biamalveiro! Since the dependency is now in setup.py, can you remove it in Pipfile and generate a new lock file please? I can do it too, if you're short on time.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "feedzai = feedzai_altair_theme.theme:feedzai",
     ],
 }
-DEPENDENCIES: List[str] = ["altair==4.*"]
+DEPENDENCIES: List[str] = ["altair==4.*", "typing-extensions==4.*"]
```
[the remark this answers is not in the document]
Comment (biamalveiro) [reply setup.py]: Of course! ✅

**Comment by biamalveiro** [comment]
Thanks for the feedback! I've made the suggested changes and generated a new lock file :)

**Comment by joaopalmeiro** [comment]
> Thanks for the feedback! I've made the suggested changes and generated a new lock file :)

Thanks, @biamalveiro! 😄 

One more thing, which I completely forgot to say, sorry! Can you also update the `CHANGELOG.md` file and bump the package version in the `__init__.py` file, please (since we need to release a new package version with this update)? Proposal for the `CHANGELOG.md` file:

```
## [1.1.0] - 2022-06-20

### Fixed

- Add `typing-extensions` as a package dependency.
```

**Review by joaopalmeiro — APPROVED** [review]

**Comment by joaopalmeiro** [comment]
It's perfect! I'll merge it, thanks again! 😄

**Pull request merged**